docs/content/Examples-Tutorials-Recipes/Column-based-access.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Column-Based Access
+permalink: /recipes/column-based-access
+category: Examples & Tutorials
+subCategory: Access control
+menuOrder: 2
+---
+
+## Use case
+
+We want to manage user access to different data depending on a database
+relationship. In the recipe below, we will manage supplier access to their
+products. A supplier can't see other supplier's products.
+
+## Data schema
+
+To implement column-based access, we will use supplier's email from
+[JSON Web Token](https://cube.dev/docs/security), and the
+[queryRewrite](https://cube.dev/docs/security/context#using-query-rewrite)
+extension point to manage data access.
+
+We have `Products` and `Suppliers` tables with `hasOne` relationship from
+products to suppliers:
+
+```javascript
+// schema/Products.js
+
+joins: {
+    Suppliers: {
+        sql: `${Suppliers}.id = ${CUBE}.supplier_id`,
+        relationship: `hasOne`
+    }
+}
```

Comment:
It might be better to show both cubes fully defined in the example above, so as to not leave any ambiguity

Reply:
Added both cubes 👌